_posts/2023-8-1-postgresql-versions-update.md

```diff
@@ -14,3 +14,9 @@ Please note that users are still able to explicitly set any of the [currently su
 ```shell
 cf create-service aws-rds <database-plan> <database-name> -c '{"version":"<desired-version>"}'
 ```
+
+For example,
+
+```shell
+cf create-service aws-rds micro-psql my-test-service -c '{"version":"15"}'
+```
```

Comment:
````suggestion
For example:

```shell
cf create-service aws-rds micro-psql my-test-service -c '{"version":"14"}'
```
````

Since the default version is currently 15, we want to show how to set an alternate version